ingestion-edge/tests/unit/conftest.py

```diff
@@ -26,6 +26,16 @@ def client() -> PublisherClient:
     return client
 
 
+@pytest.fixture(autouse=True)
+def bad_pubsub():
+    if "PUBSUB_EMULATOR_HOST" not in os.environ:
+        os.environ["PUBSUB_EMULATOR_HOST"] = "."
```

Comment:
this allows `PublisherClient` to be instantiated and fail if it's actually used (which in unit tests it isn't)